config/development.json

```diff
@@ -2,7 +2,7 @@
   "evidenceUpload": {
     "questionPage": {
       "enabled": "false",
-      "overrideAllowed": "false"
+      "overrideAllowed": "true"
```

Comment:
do we need to change this?

Reply:
So that local development is consistent with all other environments